pkgs/by-name/gi/gitlab-release-cli/package.nix

```
@@ -0,0 +1,26 @@
+{ lib
+, fetchFromGitLab
```

Needed to skip the failing test (see suggestion below)
```suggestion
, stdenv
, fetchFromGitLab
```

---

Thanks for testing on darwin